### PR TITLE
C22_WMDE_Test_23

### PR DESCRIPTION
**Pull request by moiikana**
https://phabricator.wikimedia.org/T325619

### CODE REVIEW

**Comment by moiikana** [comment]
rebased main

File: banners/desktop/components/Banner.jsx [hunk 1]
```diff
@@ -131,6 +131,12 @@ export class Banner extends Component {
 		this.props.onClose();
 	};
 
+	onGiveFeedback = e => {
+		this.onCloseBanner( e );
+		this.props.trackingData.tracker.trackBannerEvent( 'feedback-button-clicked', 0, 0, this.props.trackingData.bannerClickTrackRatio );
```
Comment (Abban) [inline banners/desktop/components/Banner.jsx:136]: There's a trackBannerEvent method in the Banner that you could use here, but it's not super important.

File: banners/desktop/translations.js [hunk 2]
```diff
@@ -18,7 +18,10 @@ const Translations = {
 	'form-step-3-error': 'Bitte wählen Sie einen Betrag aus.',
 	'address-type-notice-full': 'Sie erhalten eine Spendenquittung per Post und eine Bestätigung per E-Mail.',
 	'address-type-notice-email': 'Sie verzichten auf eine Spendenquittung, erhalten aber eine Bestätigung per E-Mail.',
-	'address-type-notice-none': 'Sie erhalten weder eine Spendenquittung noch eine Bestätigung per E-Mail.'
+	'address-type-notice-none': 'Sie erhalten weder eine Spendenquittung noch eine Bestätigung per E-Mail.',
+
+	'soft-close-button-conjunction': 'oder',
```
Comment (Abban) [inline banners/desktop/translations.js:23]: These could go into the soft close component translations.

**Review by Abban — APPROVED** [review]

File: banners/desktop/translations.js (hunk 2 above)
Comment (moiikana) [reply banners/desktop/translations.js:23]: thought the feedback button might be super temporary, that's why I rather did not put it into the shared translations

**Pull request merged**